src/note.jl

```diff
@@ -112,6 +113,11 @@ end
 
 Base.copy(notes::Notes) = Notes([copy(n) for n in notes], notes.tpq)
 
+"""
+    is_octave(note1::Note,note2::Note) -> Bool
+Determine if two notes form an octave.
```

Comment:
```suggestion
Return `true` if two notes form an octave.
```
Being explicit about the return value not only makes the documentation string more precise, but also eliminates the need of adding `-> Bool` in the first line.